packages/calcite-components/src/components/color-picker/color-picker.scss

```diff
@@ -78,11 +78,14 @@
   @apply text-n1
     focus-base
     absolute
-    z-default;
-
+    z-default
+    rounded-full
+    bg-transparent
+    w-px 
+    h-px;
   &:focus {
     @apply focus-outset;
-    outline-offset: 16px;
```

Comment:
One more thing I noticed. The design spec seems to have a 2px space between the thumb and the focus outline. The spacing in the PR screenshot looks larger than that. Maybe adjusting this to 14px can help better match that.

Reply:
if we increase the spacing from 12px to 14px it will increase the gap between the thumb and outline. 

Thoughts @ashetland

Reply:
Adding @macandcheese.

Reply:
If I’m following, I agree it should be a 2px gap to align with other “outset focus” styles.